packages/functoria-runtime/functoria-runtime.2.1.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "1.2"
+maintainer:   "Gabriel Radanne <drupyog@zoho.com>"
```

Comment:
Sure but TBH I prefer if I can ping one or two responsible persons directly when there's an issue regarding the package. With a collective address or even worse a mailing list address responsability is more diffuse and it can be more difficult to chase down people who feel responsible.